tests/functional/image/test_content.py

```diff
@@ -0,0 +1,47 @@
+import json
+import pytest
+
```

Comment:
Nit: mind adding a todo statement for `docker_history`?

Reply:
Added a few more tests that include `docker_history`, and a comment on what is missing